controller-server/src/main/java/com/yahoo/vespa/hosted/controller/LockedApplication.java

```diff
@@ -147,7 +147,7 @@ public LockedApplication withNewDeployment(ZoneId zone, ApplicationVersion appli
                                                   previousDeployment.clusterUtils(),
                                                   previousDeployment.clusterInfo(),
                                                   previousDeployment.metrics(),
-                                                  previousDeployment.activity());
+                                                  previousDeployment.activity(), Collections.emptyMap());
```

Comment:
This should be `previousDeployment.loadBalancers()` so load balancers are not thrown away when updating deployments.